presto-product-tests/conf/docker/singlenode-ldap/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+
+  presto-master:
+    image: 'teradatalabs/centos6-java8-oracle-ldap'
+    command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode-ldap run
+    volumes:
+      - ../../../../presto-product-tests/conf/tempto/tempto-configuration-for-docker-ldap.yaml:/docker/volumes/tempto/tempto-configuration-local.yaml
+    extra_hosts:
```

Comment:
yes, that's correct. Once we have openLDAP in docker, we can remove this section and run tests on Travis.